### PR TITLE
Upgrade to AdminLTE 2.1.0.

### PR DESCRIPTION
**Pull request by tim96**
Q | A
------------ | -------------
Bug fix? |	no
New feature? |	yes
BC breaks? |	no
Deprecations? |	no
Tests pass? |	yes
Fixed tickets |	#2983 
License |	MIT
Doc PR |	no
Upgrade to AdminLTE 2.1.0.
Add example of right menu to standard_layout.

### CODE REVIEW

**Comment by scolandrea** [comment]
Someone can merge this commit?

**Comment by Soullivaneuh** [comment]
You added `sonata_right_side` block but what logic did you put in this?

Can we have screenshot of the implementation results?

File: Resources/views/standard_layout.html.twig [hunk 1]
```diff
@@ -312,6 +315,60 @@ file that was distributed with this source code.
...
+
+                        <li><a href="#control-sidebar-settings-tab" data-toggle="tab"><i class="fa fa-gears"></i></a></li>
+                    </ul>
+                    <!-- Tab panes -->
+                    <div class="tab-content">
+                        <!-- Home tab content -->
+                        <div class="tab-pane active" id="control-sidebar-home-tab">
+                            <h3 class="control-sidebar-heading">{{ 'home_tab'|trans({}, 'SonataAdminBundle') }}</h3>
+                            <ul class='control-sidebar-menu'>
+                                <!-- <li>
```
Comment (Soullivaneuh) [inline Resources/views/standard_layout.html.twig]: Just copy code as sample is not good. Please cleanup code of those useless comments and give us some use cases.

**Comment by Soullivaneuh** [comment]
PR reviewed, waiting for @tim96 answer.

Thanks.

File: Resources/views/standard_layout.html.twig (hunk 1 above)
Comment (rande) [inline Resources/views/standard_layout.html.twig]: I think we can just drop this column for now. There is no use of it. we can leave the ``sonata_right_side`` block

**Comment by rande** [comment]
This PR need to be rebased.

**Comment by tim96** [comment]
@Soullivaneuh, screenshot with right sidebar https://www.dropbox.com/s/n6gk1zvdl87tw2u/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202015-06-04%2021.35.53.png?dl=0
@rande I did a small error in rebase and I forget what I change my master branch(I revert them all), what's why you see a lot of commits, sorry. I can create a new PR if you want.

**Comment by Soullivaneuh** [comment]
@tim96 

> screenshot with right sidebar

Yes, so the sidebar has nothing to show. Why introducing an empty block?

@rande what is your opinion?

> I did a small error in rebase and I forget what I change my master branch(I revert them all), what's why you see a lot of commits, sorry. I can create a new PR if you want.

you can also squash your commits: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by tim96** [comment]
@Soullivaneuh 
> Yes, so the sidebar has nothing to show. Why introducing an empty block?
Yes, it is empty block. It is like an example to show the some features from new version AdminLTE. I don't know but maybe you have roadmap or some ideas how to fill in this block.

**Comment by Soullivaneuh** [comment]
Let @rande decide about it.

**Comment by Soullivaneuh** [comment]
ping @rande for review.

@tim96 can you please squash your commits?

**Comment by greg0ire** [comment]
There are way to many commits in this PR. Looks like you rebase commits that were not yours.

**Comment by tim96** [comment]
It seems, I have an error when rebase and squash commits. Sorry it is my mistake. Maybe instead of this PR, I need create a new one?

**Comment by greg0ire** [comment]
No, please don't. Rebase interactively on master, remove the commit you did not write, then `push --force` your branch to your fork, this PR will me automagically updated.

**Comment by OskarStark** [comment]
adding the screenshot directly, in case the dropbox link will be unavailable

sidebar_screenshot

@tim96 can you please rebase it correctly, so it could be merged soon? thank you

**Comment by Soullivaneuh** [comment]
> @rande @Soullivaneuh maybe we should introduce a configration option to hide (by default) and show the sidebar

Or find a way to use it or configure block inside. Then, if no content configured, the sidebar won't be shown.

**Comment by OskarStark** [comment]
i like your idea @Soullivaneuh

**Comment by tim96** [comment]
@Soullivaneuh  I squash my commits. What about a configuration option to hide right menu?

**Comment by Soullivaneuh** [comment]
@tim96, yes. See https://github.com/sonata-project/SonataAdminBundle/pull/2992#issuecomment-123608852. ;-)

**Comment by Soullivaneuh** [comment]
Note: I think you did something wrong with your squash.

**Comment by tim96** [comment]
> @tim96, yes. See #2992 (comment). ;-)
I delete right menu example from commit, because we don't have roadmap, and I don't know which information put there. If somebody wants to add this features, he can override template.
> Note: I think you did something wrong with your squash.
@Soullivaneuh what do you mean? I have done it first time, sorry if I have a mistake. Before I never used squash command.

**Comment by Soullivaneuh** [comment]
> @Soullivaneuh what do you mean?

Just look at your commits. You have two commit with one weird merge inside. This should not be like this.

**Comment by Soullivaneuh** [comment]
`2.2.0` is out: https://github.com/almasaeed2010/AdminLTE/tree/v2.2.0

Maybe could you take a look?

**Comment by tim96** [comment]
> Maybe could you take a look?

I have installed version 2.2 for one of my project yesterday, it seems what all functions work. Maybe I can create a new PR for upgrade to AdminLTE 2.2, @Soullivaneuh ?

**Comment by greg0ire** [comment]
I also think a new PR would be best, because of the branch name you chose (it will show up in the merge commit message, which could be confusing).

**Comment by Soullivaneuh** [comment]
@greg0ire @tim96 Agreed. Let's got for a new PR, if you want to continue your work of course. :+1:

**Comment by tim96** [comment]
> You have two commit with one weird merge inside. 

I understand my error, sorry.

> @greg0ire @tim96 Agreed. Let's got for a new PR, if you want to continue your work of course. :+1:

@Soullivaneuh , I have created a new PR https://github.com/sonata-project/SonataAdminBundle/pull/3142

**Comment by Soullivaneuh** [comment]
Ok closing this one so.

**Pull request closed without merging**